deploy/kubernetes/elastic-agent-standalone-kubernetes.yml

```
@@ -215,10 +370,10 @@ spec:
       serviceAccountName: elastic-agent
       containers:
         - name: elastic-agent
-          image: docker.elastic.co/beats/elastic-agent:7.12.0-SNAPSHOT
+          image: docker.elastic.co/beats/elastic-agent:8.0.0-SNAPSHOT
```

Use `%VERSION%` here and in other tags.

```suggestion
          image: docker.elastic.co/beats/elastic-agent:%VERSION%
```